src/hera/workflows/_mixins.py

```diff
@@ -493,7 +493,13 @@ def _build_inputs(self) -> Optional[ModelInputs]:
         return inputs
 
 
-class TemplateInvocatorMixin(BaseMixin):
+class TemplateInvocatorSubNodeMixin(BaseMixin):
```

Comment:
FYI @flaviuvadan / @samj1912 I want to rename this mixin to be accurate before the official release